lib/report_formatter/timeline.rb

```diff
@@ -106,12 +106,21 @@ def tl_event(tl_xml, row, col)
             ems = ExtManagementSystem.find(rec[:ems_id])
             ems_cloud =  true if ems.kind_of?(EmsCloud)
           end
-          if rec[:vm_name] && !ems_cloud             # Create the title using VM name
-            e_title = rec[:vm_name]
-          elsif rec[:host_name] && !ems_cloud                 #   or Host Name
-            e_title = rec[:host_name]
-          elsif rec[:ems_cluster_name] && !ems_cloud          #   or Cluster Name
-            e_title = rec[:ems_cluster_name]
+          if !ems_cloud
+            if rec[:vm_name]                           # Create the title using VM name
+              e_title = rec[:vm_name]
+            elsif rec[:host_name]                      #   or Host Name
+              e_title = rec[:host_name]
+            elsif rec[:ems_cluster_name]               #   or Cluster Name
+              e_title = rec[:ems_cluster_name]
+            elsif rec[:container_name]
+              e_title = rec[:container_name]
+            elsif rec[:container_group_name]
+              e_title = rec[:container_group_name]
+            elsif rec[:container_replicator_name]
+              e_title = rec[:container_replicator_name]
+            elsif rec[:container_node_name]
+              e_title = rec[:container_node_name]
```

Comment:
@dkorn `end` is missing for the `if` block. Also wondering if this can be changed to 

e_title =  if rec[:vm_name]                           # Create the title using VM name
             rec[:vm_name]
           elsif rec[:host_name]                      #   or Host Name
             rec[:host_name]
           elsif rec[:ems_cluster_name]               #   or Cluster Name
             rec[:ems_cluster_name]
           elsif rec[:container_name]
             rec[:container_name]
           elsif rec[:container_group_name]
             rec[:container_group_name]
           elsif rec[:container_replicator_name]
             rec[:container_replicator_name]
           elsif rec[:container_node_name]
             rec[:container_node_name]
           end